nifi-commons/nifi-utils/src/test/java/org/apache/nifi/util/validator/TestStandardValidators.java

```diff
@@ -184,6 +184,25 @@ public void testListValidator() {
         assertFalse(vr.isValid());
         assertEquals(1, mockValidator.getValidateCallCount());
 
+        // An empty list is the same as null, "" or " "
+        vr = val.validate("List", ",", validationContext);
+        assertFalse(vr.isValid());
+        assertEquals(0, mockValidator.getValidateCallCount());
+
+        vr = val.validate("List", " , ", validationContext);
+        assertFalse(vr.isValid());
+        assertEquals(1, mockValidator.getValidateCallCount());
+
+        // will evaluate to no entry
+        vr = val.validate("List", ",,,,", validationContext);
+        assertFalse(vr.isValid());
+        assertEquals(0, mockValidator.getValidateCallCount());
+
+        // will evaluate to an empty element
+        vr = val.validate("List", ",foo", validationContext);
```

Comment:
Since this list has one non-empty element, should it be valid?

Reply:
No.  The way it works is each item in a produced list is passed to the embedded validator.
In this case, the split produces ["","foo"].   2 elements.
The first element is passed to the non-empty validator and fails.

Reply:
Ah I see, it's because you're using a non-empty validator for each element.

Reply:
Yes, that is how the tests were already